<?xml version="1.0"?>
<launch>
    <arg name="threhold_of_distance_btw_pc_and_person" default="0.35"/>
    <arg name="range" default="6.0"/>
    <arg name="grid_num" default="900"/>
    <arg name="grid_width" default="30"/>
	<arg name="people_num" default="30"/>
    <arg name="flow_image_size" default="30"/>
    <arg name="hz" default="10.0"/>
    <arg name="image_hz" default="100.0"/>
    <arg name="is_save_image" default="false"/>
    <arg name="save_data_num" default="1"/>
	<arg name="pkg_path" default="/home/amsl/ros_catkin_ws/src/bev_evaluator/bev_img"/>

	<param name="use_sim_time" value="true"/>

	<node pkg="bev_evaluator" type="bev_evaluator" name="bev_evaluator" output="screen">
        <param name="THREHOLD_OF_DISTANCE_BTW_PC_AND_PERSON" value="$(arg threhold_of_distance_btw_pc_and_person)" type="double"/>
		<param name="RANGE" value="$(arg range)" type="double"/>
		<param name="GRID_NUM" value="$(arg grid_num)" type="int"/>
        <param name="GRID_WIDTH" value="$(arg grid_width)" type="int"/>
		<param name="PEOPLE_NUM" value="$(arg people_num)" type="int"/>
		<param name="FLOW_IMAGE_SIZE" value="$(arg flow_image_size)" type="int"/>
		<param name="Hz" value="$(arg hz)" type="double"/>
		<param name="SAVE_NUMBER" value="$(arg save_data_num)" type="int"/>
        <!-- <rosparam command="load" file="$(find cats)/config/cats_motion_param.yaml"/> -->
	</node>

	<!-- optional -->
	<!-- <include file="$(find bev_evaluator)/launch/bag_player.launch"/> -->
	<include file="$(find velodyne_height_map)/launch/amsl_velodyne_heightmap.launch"/>
	<include file="$(find velodyne_pointcloud)/launch/32e_points.launch"/>

</launch>
